### ISSUE TITLE
REFramework Caused Crash Even After Using Fresh Download

### ISSUE THREAD

**Issue by yudster13** [praydog/REFramework#1035]
**Describe the bug**
So the title pretty much says it all. I used REFramework just for the free cam tool (also activated the feature to deinterlaced the graphics once you're scoping in on a Sniper Rifle). The game would crash unpredictably. Sometimes in a matter of minutes, sometimes after I've been playing for so many hours. Weirdly, I once tried to remove the mod and did a fresh download of the game but the issue still persist. This time, the Capcom Crash Tool shows up. But the game would not crash until after I press "Close" on the Capcom Crash Tool.

I've reinstalled the mod and tried to recreate the issue. This time, the crash happened appear after I've been playing for approximately 2 hours. I've also uploaded the dmp file.
[reframework_crash.dmp](https://github.com/praydog/REFramework/files/14944394/reframework_crash.dmp)

I can't upload the txt file since it's too big (72 mb). Any other way to upload it?

**Identify the REFramework version**
I've used the REFramework I downloaded from Nexus Mod. I believe it was the Github developer build 8265afb
https://www.nexusmods.com/residentevil42023/mods/12?tab=files

**Desktop and VR (please complete the following information):**
 - Windows 11
 - RTX 3070 Ti 8GB
 - Ryzen 7 5700x

**Comment by yudster13** [comment]
This is a new .dmp and .txt file
[Uploading re2_framework_log.txt…]()

**Comment by praydog** [comment]
Can you clarify for me? Did you say this crash also happened with the mod removed? That is, removing dinput8.dll from your game directory?

**Comment by yudster13** [comment]
> Weirdly, I once tried to remove the mod and did a fresh download of the game but the issue still persist. This time, the Capcom Crash Tool shows up. But the game would not crash until after I press "Close" on the Capcom Crash Tool.

Yep. I did a clean uninstall of the game through Steam, and deleting the dinput8.dll also the leftover RE4 folder.
At least, I thought the mod is the cause of the problem because I never have this happened before. RE4 used to run smoothly without crashing before and it is currently the only game in my PC right now that is crashing.

**Comment by yudster13** [comment]
Hey there, didn't mean to rush but do you have any update on this?

**Comment by praydog** [comment]
No. You said the crash was happening without REFramework installed. REFramework does not leave anything that could affect the game if dinput8.dll is removed.

**Comment by yudster13** [comment]
> No. You said the crash was happening without REFramework installed. REFramework does not leave anything that could affect the game if dinput8.dll is removed.

Okay, but that doesn't fix the original issue that I'm also asking though? The game is still crashing with the mod on. I should add that these crashing never occured before, only after I used the mod and right now the game is still crashing with the mod on.

**Comment by 7516Krishna** [comment]
Hey, I've downloaded Fluffy Manager with Mods and Ultimate Ammo with Script Core and Reframework. But my game crashes. Even after deleting Reframework, the game starts with mods enabled, but I'm stuck on a black screen with no start screen - just game sounds in the background .
